acir/src/native_types/arithmetic.rs

```diff
@@ -149,7 +151,16 @@ impl Expression {
         Ok(expr)
     }
 
-    // TODO: possibly rename, since linear can have one mul_term
+    /// Returns true if highest degree term in the expression is one.
```

Comment:
No, the function returns true if there is no degree 2 term.

Reply:
Updated